SPECS/binutils/binutils.spec

```diff
@@ -44,6 +46,9 @@ find %{buildroot} -type f -name "*.la" -delete -print
 rm -rf %{buildroot}%{_infodir}
 %find_lang %{name} --all-name
 
+install -m 644 libiberty/libiberty.a %{buildroot}%{_libdir}
+install -m 644 include/libiberty.h %{buildroot}%{_includedir}
```

Comment:
Double checking here; is `libiberty.h` the only header which we need to install or there is something else also ?

Reply:
Just the header and the static lib.